### DISCUSSION TITLE
install nivo ScatterPlot chart

### DISCUSSION THREAD

**Discussion by janakapdj** [plouc/nivo#2210] [Q&A]
I am trying to use nivo ScatterPlot chart with Storybook 
but the installation keeps giving missing dependency error for 

node_modules/@nivo/scatterplot/node_modules/@nivo/annotations/dist/nivo-annotations.es.js
Attempted import error: 'useAnimatedPath' is not exported from '@nivo/core'

but I already have dependency installed
    "@nivo/annotations": "^0.80.0",
    "@nivo/core": "^0.80.0",
    "@nivo/line": "^0.61.1",
    "@nivo/radar": "^0.61.1",
    "@nivo/scatterplot": "^0.80.0",
    "@nivo/sunburst": "^0.61.0",
    "@nivo/tooltip": "^0.80.0",
    
    can anyone suggest if is there anything that needs to be changed??

**Comment by petrvecera** [comment]
I am not sure if it's OK to have charts version different than the core. You are mixing versions 0.61 and 0.80. Might not be culprit directly but I think it's not he best idea.

**Comment by plouc** [comment]
As @petrvecera guessed, mixing versions is not supported, all versions should be aligned.
